lib/kb_ObjectInfo/CreateFeatureLists_Report.py

```diff
@@ -539,11 +534,11 @@ def readFeatSeq(self, pyStr, format):
 #   Type Unknown
 #
         else:
-            print ("This type of FeatureSet has not been described yet")
+            log("This type of FeatureSet has not been described yet")
```

Comment:
Good point. This can be easily resolved. To be fair though, the only way that this would be triggered is if bad data passed or a new object definition was used. Neither of these is a user issue and putting it the report won't be useful to a user. It will be added to the list of improvements because it isn't urgent.